src/main.js

```diff
@@ -110,7 +114,17 @@ class Entry {
  * @returns {Array} An array containing entry objects.
  */
 const parseVarFromType = envSpecAsArray => {
-  return (envSpecAsArrayParsed = envSpecAsArray.map(element => {
+  //in case a line started with "#" we would have a blank environmental variable
+  envSpecAsArray = envSpecAsArray.filter(line => line[0] !== "#");
```

Comment:
First, we should avoid reassigning values to arguments inside a function. You can just:

```javascript
return envSpecAsArray.filter(...).map(...)
```

Next, we can just use the [`startswith` JavaScript function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) instead of this manual check 😄.